wiki/Beatmapping/Circle_Size/en.md

```diff
@@ -0,0 +1,32 @@
+---
+tags:
+  - CS
+---
+
+# Circle size
+
+Circle size, commonly abbreviated as *CS*, is a difficulty setting which determines the size of [hit objects](/wiki/Hit_Objects) on [beatmaps](/wiki/Beatmaps). The value for circle size ranges from 2 to 7; where a higher value will make hit objects smaller. Most beatmaps use a value between 3 and 5, while values between 5 and 6 are sometimes used. Values between 6 and 7 are almost never used. Circle size values higher than 7 or lower than 2 can be seen when the [Hard Rock](/wiki/HR) or [Easy](/wiki/EZ) mods are used respectively.
+
+It is possible to set the circle size value outside normal bounds by manually editing the [.osu file](/wiki/osu!_File_Formats/Osu_(file_format)). However, this is considered unrankable by the [ranking criteria](/wiki/Ranking_Criteria).
+
+## osu!standard
+
+In osu!standard, only the size of hit circles and sliders are adjusted through CS, while spinners remain unaffected. Mathematically, circle size is derived through the following formula:
```

Comment:
```suggestion
In osu!standard, only the size of hit circles and sliders are adjusted through circle size, while spinners remain unaffected. Mathematically, circle size is derived through the following formula:
```

I just noticed that in the other game mode sections, they actually spell out circle size but this one only uses CS